app/scripts/router.js

```diff
@@ -89,6 +89,7 @@ function (
       'cannot_create_account(/)': showView(CannotCreateAccountView),
       'verify_email(/)': showView(CompleteSignUpView),
       'confirm(/)': showView(ConfirmView),
+      'settings?setting=avatar': showView(AvatarChangeView),
```

Comment:
Ahha, I see..  This requires a backbone update. With the updated backbone, the route matches, but only without a `/` after settings. If I add a `(/)` before the `?`, both routes match.